### PR TITLE
fix: add missing translation fallbacks

### PR DESCRIPTION
**Pull request by henriettemoe**

### CODE REVIEW

File: .storybook/preview.tsx [hunk 1]
```diff
@@ -39,17 +41,23 @@ const translations = Object.fromEntries(
   }),
 );
 
+const l10n = { ...defaultTranslations, ...translations };
+
 const h5pInstance = new H5PWrapper(
   {
     behaviour: {},
     // @ts-ignore
```
Comment (boyum) [inline .storybook/preview.tsx:49]:
```suggestion
```

File: .storybook/preview.tsx [hunk 2]
```diff
@@ -39,17 +41,23 @@ const translations = Object.fromEntries(
...
-    l10n: {},
+    l10n: l10n,
```
Comment (boyum) [inline .storybook/preview.tsx:50]:
```suggestion
    l10n,
```

File: .storybook/preview.tsx [hunk 3]
```diff
@@ -39,17 +41,23 @@ const translations = Object.fromEntries(
   }),
 );
 
+const l10n = { ...defaultTranslations, ...translations };
```
Comment (boyum) [inline .storybook/preview.tsx:44]: I guess `translations` is now obsolete? Won't it be the same as `defaultTranslations`, and just for mock data anyways?

```suggestion
const l10n = defaultTranslations;
```

File: src/utils/semantics.utils.ts [hunk 4]
```diff
@@ -113,6 +113,6 @@ export const getEmptyParams = (): Required<Params> => {
       colorTheme: defaultTheme,
     },
     behaviour: null,
-    l10n: {} as Translations,
```
Comment (boyum) [inline src/utils/semantics.utils.ts:116]: Is this used for anything?

Comment (henriettemoe) [reply src/utils/semantics.utils.ts:116]: Don't seem like the l10n is used directly from params, but it doesn't hurt to make sure it is sanitized there as well

**Review by boyum — APPROVED** [review]

File: src/utils/semantics.utils.ts (hunk 4 above)
Comment (henriettemoe) [reply src/utils/semantics.utils.ts:116]: getEmptyParams is used in H5PWrapper to ensure fallback for the params like so:
{
  ...getEmptyParams(),
  ...params,
};

**Pull request merged**